src/Xhgui/Searcher/Pdo.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+class Xhgui_Searcher_Pdo implements Xhgui_Searcher_Interface
+{
+    /**
+     * @var Xhgui_Saver_Pdo
+     */
+    private $saver;
+
+    /**
+     * @var PDO
+     */
+    private $pdo;
+
+    /**
+     * @var string
+     */
+    private $table;
+
+    /**
+     * @param Xhgui_Saver_Pdo $saver
+     * @param PDO             $pdo   An open database connection
+     * @param string          $table Table name where Xhgui profiles are stored
+     */
+    public function __construct(Xhgui_Saver_Pdo $saver, PDO $pdo, $table)
+    {
+        $this->saver = $saver;
+        $this->pdo = $pdo;
+        $this->table = $table;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function latest()
+    {
+        $stmt = $this->pdo->query("
+          SELECT
+            id,
+            profile,
+            url,
+            SERVER,
+            GET,
+            ENV,
+            simple_url,
+            request_ts,
+            request_ts_micro,
+            request_date
+          FROM {$this->table}
+          ORDER BY request_date ASC
+          LIMIT 1
+        ");
+
+        if (false === $row = $stmt->fetch(PDO::FETCH_ASSOC)) {
+            throw new Exception('No profile available yet.');
+        }
```

Comment:
Should we use an exception subclass? That would let is render different/better error pages.

Reply:
Maybe, but for the moment I just did the same that `Xhgui_Profiles` (err, `Xhgui_Searcher_Mongo`) already did in the `_wrap()` method. Personally I'd save this item for a separate PR. This one is sprawling enough.